Cargo.toml

```diff
@@ -47,20 +47,24 @@ lazy_static = "1.4.0"
 mime_guess = "2.0.4"
 regex = "1.7.1"
 rocksdb = "0.20.1"
-serde_json = "1.0.94"
+serde_json = "1.0.117"
 tempfile = "3.4.0"
 tracing = "0.1"
 uhlc = "0.5.2"
 walkdir = "2.3.2"
-zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main", features = ["unstable"] }
-zenoh-codec = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-collections = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-core = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-protocol = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-util = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh_backend_traits = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-plugin-trait = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-
+zenoh = { git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", features = [
+    "unstable",
+    "internal",
+] }
+zenoh-buffers = { git = "https://github.com/eclipse-zenoh/zenoh", branch = "dev/1.0.0" }
```

Comment:
In `dev/1.0.0` you should not be forced to import anything else than `zenoh` crate. Could you try to remove all crates excepts for `zenoh_backend_traits` and `zenoh-plugin-trait`?